src/application/extension-popup-menu/footer.tsx

```diff
@@ -0,0 +1,46 @@
+import { Icon, IconName } from 'shared/ui';
+
+interface FooterLink {
+  iconName: IconName;
+  href: string;
+  text: string;
+}
+
+export const Footer = () => {
+  const footerLinks: FooterLink[] = [
+    {
+      href: 'https://docs.idriss.xyz/user-guides/idriss-book#where-can-you-use-the-extension',
+      iconName: 'FileTextIcon',
+      text: 'Docs',
+    },
+    {
+      href: 'https://discord.gg/RJhJKamjw5',
+      iconName: 'DiscordLogoIcon',
+      text: 'Discord',
+    },
+    {
+      href: 'https://twitter.com/IDriss_xyz',
+      iconName: 'TwitterLogoIcon',
+      text: 'Twitter',
+    },
+  ];
```

Comment:
this thing is static so could be moved outside of component (ideally constants file), interface also won't be needed since you're not expecting it as a prop but just consuming it directly in JSX

Reply:
I've moved the `footerLinks` to constants.ts along with the interface. I think we can keep it this way to make it easier to edit the `iconName` props in the future